redirect/README.md

```diff
@@ -0,0 +1,145 @@
+
+```go
+client := redirect.NewAPIClient(option.WithAPIKey("YOUR_API_KEY"))
+```
+
+You can also set the API key using the environment variable `BLOXONE_API_KEY`
+
+Note: The API key is a secret and should be handled securely. Hardcoding the API key in your code is not recommended.
+
+### Default Tags
+
+You can set default tags for all API requests using the `option.WithDefaultTags` option. For example:
+
+```go
+client := redirect.NewAPIClient(option.WithDefaultTags(map[string]string{"tag1": "value1", "tag2": "value2"}))
+```
+This will add the tags `tag1=value1` and `tag2=value2` to all API requests that support tags in the request body.
+
+## Documentation for API Endpoints
+
+All URIs are relative to *https://csp.infoblox.com/api/atcfw/v1*
+
+Class | Method | HTTP request | Description
+------------ | ------------- | ------------- | -------------
+*CertificateAPI* | [**GetProxyCertificates**](docs/CertificateAPI.md#getproxycertificates) | **Get** /cert_download_urls | Get Proxy Certificates
+*CustomRedirectsAPI* | [**CreateCustomRedirect**](docs/CustomRedirectsAPI.md#createcustomredirect) | **Post** /custom_redirects | Create Custom Redirect.
+*CustomRedirectsAPI* | [**DeleteCustomRedirect**](docs/CustomRedirectsAPI.md#deletecustomredirect) | **Delete** /custom_redirects | Delete Custom Redirect.
+*CustomRedirectsAPI* | [**DeleteSingleCustomRedirect**](docs/CustomRedirectsAPI.md#deletesinglecustomredirect) | **Delete** /custom_redirects/{id} | Delete Custom Redirect By Id.
+*CustomRedirectsAPI* | [**ListCustomRedirect**](docs/CustomRedirectsAPI.md#listcustomredirect) | **Get** /custom_redirects | List Custom Redirects.
+*CustomRedirectsAPI* | [**ReadCustomRedirect**](docs/CustomRedirectsAPI.md#readcustomredirect) | **Get** /custom_redirects/{id} | Read Custom Redirect.
+*CustomRedirectsAPI* | [**UpdateCustomRedirect**](docs/CustomRedirectsAPI.md#updatecustomredirect) | **Put** /custom_redirects/{id} | Update Custom Redirect.
+*RedirectPageAPI* | [**ReadRedirectPage**](docs/RedirectPageAPI.md#readredirectpage) | **Get** /redirect_page | Read Redirect Page.
+*RedirectPageAPI* | [**UpdateRedirectPage**](docs/RedirectPageAPI.md#updateredirectpage) | **Put** /redirect_page | Update Redirect Page.
+
+
+## Documentation For Models
+
+ - [CertificateGetProxyCertificates500Response](docs/CertificateGetProxyCertificates500Response.md)
+ - [CertificateGetProxyCertificates500ResponseError](docs/CertificateGetProxyCertificates500ResponseError.md)
+ - [CustomRedirect](docs/CustomRedirect.md)
+ - [CustomRedirectCreateResponse](docs/CustomRedirectCreateResponse.md)
+ - [CustomRedirectDeleteRequest](docs/CustomRedirectDeleteRequest.md)
+ - [CustomRedirectMultiResponse](docs/CustomRedirectMultiResponse.md)
+ - [CustomRedirectReadResponse](docs/CustomRedirectReadResponse.md)
+ - [CustomRedirectUpdateResponse](docs/CustomRedirectUpdateResponse.md)
+ - [CustomRedirectsCreateCustomRedirect400Response](docs/CustomRedirectsCreateCustomRedirect400Response.md)
```

Comment:
The `...Response` models didn't get renamed. Maybe the model name mappings script doesn't account for that. Something to be fixed i guess.